README.md

```diff
@@ -26,12 +26,17 @@ Please click [here](https://github.com/csterryliu/Legal-Attest-Letter-Generator-
 ## How To Use It ##
 Please read [Wiki](https://github.com/csterryliu/Legal-Attest-Letter-Generator-TW/wiki/).
 
-## Dependency ##
-- [PyPDF2 1.26](https://github.com/mstamy2/PyPDF2)  
-- [reportlab 3.3.0](https://bitbucket.org/rptlab/reportlab)
+## Dependencies ##
+- [PyPDF2 3.0.1](https://pypi.org/project/PyPDF2/)
+- [reportlab 4.0.9](https://pypi.org/project/reportlab/)
```

Comment:
Since we are using `requirements.txt` to maintain the dependency versions, I'd like to point these two libraries back to pypi.org